src/Media_Command.php

```diff
@@ -947,7 +946,7 @@ private function get_registered_image_sizes() {
 		global $_wp_additional_image_sizes;
 
 		$image_sizes         = array();
-		$default_image_sizes = get_intermediate_image_sizes();
+		$default_image_sizes = array( 'thumbnail', 'medium', 'medium_large', 'large' );
```

Comment:
Can you add a Behat test that triggers this bug, to make sure that:
- we're fixing the bug we want to fix
- we won't reintroduce the bug later as a regression